src/components/Dashboard/Modals/EditAppModal/EditAppModal.tsx

```diff
@@ -90,6 +91,9 @@ export const EditAppModal = ({
       return;
     }
 
+    values.url = removeTrailingSlash(values.url);
+    values.behaviour.externalUrl = removeTrailingSlash(values.behaviour.externalUrl);
```

Comment:
Isnt this the external URL? So this would cause the same issue as #777

Reply:
Good catch, Sorry and thank you for pointing it out.